loki/transform/fortran_c_transform.py

```diff
@@ -64,7 +65,10 @@ def transform_file(self, sourcefile, **kwargs):
             self.transform_subroutine(routine, **kwargs)
```

Comment:
The `transform_file` method is redundant and should be removed. Would you mind including this here, as part of the initial house keeping?